index.js

```diff
@@ -16,6 +16,15 @@ function createWindow() {
                         app.quit();
                     }
                 }
+            ],
+            label: 'Tools',
+            submenu: [
+                {
+                    label: 'DevTools',
+                    click() {
+                        window.webContents.openDevTools()
```

Comment:
We should also be able to hide them via the menu bar, and display the keyboard shortcut next to the label.